jekyll/_cci2/android-machine-image.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: classic-docs
+title: "Using Android Images with the Machine Executor"
+short-title: "Android Image on the Machine Executor"
+description: "Using the Android Image on the Machine Executor"
+version:
+  - Cloud
+---
+
+## Overview
+
+The Android machine image is accessed through the [Linux `machine`
+executor]({{site.baseurl}}/2.0/configuration-reference/#machine-executor-linux),
+like other Linux machine images on CircleCI. The Android machine image supports
+nested virtualization and x86 Android emulators, so it can be used for Android
+UI testing. It also comes with the Android SDK pre-installed.
+
+## Using the Android machine image
+
+It is possible to configure the use of the Android image in your configuration
+with [orbs]({{site.baseurl}}/2.0/orb-intro) as well as manually. Using the
+Android orb will simplify your configuration while more complex and custom
+configurations may benefit from manually configuring your usage. This document
+will cover both use cases. Please view the [examples](#examples) section
+below for more details.
+
+## Pre-installed Software
+
+The Android machine image comes with the following pre-installed:
+
+### Android SDK
+- sdkmanager
+- Android platform 27, 28, 29, 30
+- Build tools 30.0.3
+- emulator, platform-tools, tools
+- NDK (Side-by-side) 21.4.7075529
+- cmake 3.6.4111459
+- extras;android;m2repository, extras;google;m2repository, extras;google;google_play_service
+
+### Others
+- gcloud
+- OpenJDK 8, OpenJDK 11 (default)
+- maven 3.6.3, gradle 6.8, ant
+- nodejs 12.20.1, 14.15.4 (default), 15.6.0
+- python 2.7.18, python 3.9.1
+- ruby 2.7.1, ruby 3.0.0
+- docker 20.10.2, docker-compose 1.28.2
+- jq 1.6
+
+## Limitations
```

Comment:
What do you think of removing most of the text here as it's mainly relevant to the image during the preview phase?

Maybe we could just have `There may be up to 2 mins of spin-up time before your job starts running. This
  time will decrease as more customers start using the Android image.` ?